src/group/libp2p_ack_stream.erl

```diff
@@ -53,23 +53,24 @@ init(client, Connection, [AckRef, AckModule, AckState]) ->
 
 handle_data(_Kind, Data, State=#state{ack_ref=AckRef, ack_module=AckModule, ack_state=AckState}) ->
     case libp2p_ack_stream_pb:decode_msg(Data, libp2p_ack_frame_pb) of
-        #libp2p_ack_frame_pb{data=Bin, seq=[Seq]} when Bin /= <<>> ->
+        #libp2p_ack_frame_pb{messages=[Bin], seqs=[Seq]} when Bin /= <<>> ->
```

Comment:
Presumably these becoming a list are so we can do batching next?

Reply:
yes.